apps/web/playwright/icons.e2e.ts

```diff
@@ -5,6 +5,7 @@ import { test } from "./lib/fixtures";
 test("Icons render properly", async ({ page }, testInfo) => {
   await page.goto("/icons");
   await expect(page).toHaveScreenshot("icons.png", {
+    maxDiffPixels: 50000, // Allow some pixel differences (e.g., linux vs mac)
```

Comment:
Because my local env is Mac, a slightly different sized snapshot gets generated. And we get this error:
```
 1) [@***com/web] › apps/web/playwright/icons.e2e.ts:5:5 › Icons render properly ──────────────────

    Error: Screenshot comparison failed:

      Expected an image 1265px by 1320px, received 1280px by 1320px. 41642 pixels (ratio 0.03 of all image pixels) are different.
```

If we don't like this approach, I need to run a linux container and generate the snapshot, but I don't think it's worth the effort.

Reply:
Yeah not sure it's worth a hassle.

@zomars Where did you originally generate this from?

Reply:
I pushed a better fix: https://github.com/calcom/cal.com/pull/17841/commits/1055aa7df137a520d532647190d121ddf43a3ec3